ffi/Cargo.toml

```diff
@@ -20,7 +20,7 @@ rust-version.workspace = true
 crate-type = ["staticlib"]
 
 [dependencies]
-libc = "0.2.2"
+libc = "0.2.174"
```

Comment:
nit: Did this sneak in on purpose? A quick `git grep` shows we don't use libc anywhere anyway, so maybe we just drop this.

Reply:
this was added on merge from main, updated here: 
https://github.com/ava-labs/firewood/commit/158a2d41497ea03945d8d10fd744684f897b6f9d#diff-41faf4451a7437947ddd40e9f89c46c1b8ff3de6002201c26f1f8caa598a92abL22

It shows on the merge because I to manually resolve the conflict due to touching the next line. 

We probably don't need to explicitly add it (https://github.com/ava-labs/firewood/issues/1032).